### PR TITLE
Replaces sleep() with addtimer() in decon unit, and minor tweak

### PR DESCRIPTION
**Pull request by warface1234455**
Addtimer() is better

# Document the changes in your pull request
Replaces sleep() with addtimer() in decon unit
Adds missing opening sound

# Changelog

:cl:  
tweak: Replaces sleep() with addtimer() in decon unit
soundadd: Adds missing opening sound
/:cl:

### CODE REVIEW

File: code/game/machinery/decontamination.dm [hunk 1]
```diff
@@ -117,41 +118,49 @@
...
+	for(var/obj/item/item in contents)	
+		QDEL_NULL(item)
+	shock()
+	open_machine(0)
+	if(occupant)
+		dump_mob()
+
+/obj/machinery/decontamination_unit/proc/decon_eject2()
+	var/mob/living/mob_occupant = occupant
+	say("The decontamination process is completed, thank you for your patient.")
```
Comment (ReddicusDragon) [inline code/game/machinery/decontamination.dm:146]: please update "patient" to "patience"

**Review by ReddicusDragon — CHANGES_REQUESTED** [review]

File: code/game/machinery/decontamination.dm (hunk 1 above)
Comment (ShoziOffical) [reply code/game/machinery/decontamination.dm:146]: I will put this in my other pr

Comment (ReddicusDragon) [reply code/game/machinery/decontamination.dm:146]: link me this pr when you submit it, brother

File: code/game/machinery/decontamination.dm [hunk 2]
```diff
@@ -117,41 +118,49 @@
...
-				things_to_clear += contents
-			for(var/am in things_to_clear) //Scorches away blood and forensic evidence, although the SSU itself is unaffected
-				var/atom/movable/dirty_movable = am
-				dirty_movable.wash(CLEAN_ALL)
-		open_machine(0)
-		if(occupant)
-			dump_mob()
+			addtimer(CALLBACK(src, .proc/decon_eject2), 12)
+
+/obj/machinery/decontamination_unit/proc/decon_eject1()
```
Comment (ReddicusDragon) [inline code/game/machinery/decontamination.dm]: recommend renaming this proc decon_eject_emag

File: code/game/machinery/decontamination.dm [hunk 3]
```diff
@@ -117,41 +118,49 @@
...
+		visible_message(span_warning("[src]'s gate creaks open with a loud whining noise."))
+	playsound(src, 'sound/machines/airlock_alien_prying.ogg', 50, TRUE)
+	for(var/obj/item/item in contents)	
+		QDEL_NULL(item)
+	shock()
+	open_machine(0)
+	if(occupant)
+		dump_mob()
+
+/obj/machinery/decontamination_unit/proc/decon_eject2()
```
Comment (ReddicusDragon) [inline code/game/machinery/decontamination.dm]: recommend renaming this proc just decon_eject

**Review by ReddicusDragon — CHANGES_REQUESTED** [review]

File: code/game/machinery/decontamination.dm (hunk 2 above)
Comment (ShoziOffical) [reply code/game/machinery/decontamination.dm]: I thought so yeah

File: code/game/machinery/decontamination.dm (hunk 1 above)
Comment (ShoziOffical) [reply code/game/machinery/decontamination.dm:146]: #16755

**Review by ReddicusDragon — APPROVED** [review]

**Review by JamieD1 — APPROVED** [review]

**Comment by JamieD1** [comment]
Conflicts

**Comment by ShoziOffical** [comment]
wha

**Comment by ShoziOffical** [comment]
fixed cornflake

**Comment by ShoziOffical** [comment]
@JamieD1

**Pull request merged**